docs/how-to/index.md

```diff
@@ -6,3 +6,4 @@ The how-to guides provide practical steps for working with ehrQL in your project
 * [How to use dummy data in an ehrQL measures definition](dummy-measures-data.md)
 * [How to assign multiple columns to a dataset programmatically](assign-multiple-columns.md)
 * [How to work with codelists](codelists.md)
+* [How to test the ehrql logic in your dataset definition](test-dataset-definition.md)
```

Comment:
Oh, this was the original _longer_ title, will fix this together with suggestions that come up during code review